backend/devices/bitbox/pairing.go

```diff
@@ -46,13 +46,30 @@ func (device *Device) finishPairing(channel *relay.Channel) {
 	device.fireEvent(EventPairingSuccess, nil)
 }
 
+func (device *Device) handlePairingError(err error, status string, message string) {
+	switch status {
+	case "pullFailed":
+		device.log.WithError(err).Warningf("Failed to pull the mobile's %s.", message)
```

Comment:
`Errorf` sounds more appropriate.

Maybe don't log the actual message, as try to avoid logging sensitive info